waspc/cli/src/Wasp/Cli/Command/BashCompletion.hs

```diff
@@ -26,7 +26,7 @@ bashCompletion = do
     ["db", cmdPrefix] -> listMatchingCommands cmdPrefix dbSubCommands
     _ -> liftIO . putStrLn $ ""
   where
-    commands = ["new", "version", "start", "db", "clean", "build", "telemetry", "deps"]
+    commands = ["new", "version", "waspls", "start", "db", "clean", "build", "telemetry", "deps", "info"]
```

Comment:
We also have `completion` command, you can also add that one! And `completion:generate`. These are not mentioned but they exist, check them here, @h4r1337 just added the docs for them: https://github.com/wasp-lang/wasp/pull/766/files .

Reply:
Added!  Thank you.  :bow: